### PR TITLE
Validations and model specs

### PR DESCRIPTION
**Pull request by nunegabriel**
In this milestone I did the following

- [ ] I wrote unit tests for each model
- [ ] I added all the required validations

### CODE REVIEW

File: spec/models/comment_spec.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+    it 'requires the presence of text' do
+      comment.text = nil
+      expect(comment).to be_invalid
+    end
+
+    it 'requires the presence of post_id' do
+      comment.post_id = nil
+      expect(comment).to be_invalid
+    end
```
Comment (Deepakdanger) [inline spec/models/comment_spec.rb]: - [ ] Kindly create some positive test also, ex:- if comment.text = "something", it should be valid

File: spec/models/like_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+    it 'requires the presence of author_id' do
+      like.author_id = nil
+      expect(like).to be_invalid
+    end
+
+    it 'requires the presence of post_id' do
+      like.post_id = nil
+      expect(like).to be_invalid
+    end
```
Comment (Deepakdanger) [inline spec/models/like_spec.rb:15]: - [ ] Kindly create some positive test also,

File: spec/models/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  subject { described_class.new(title: 'deadline day', comments_counter: 4, likes_counter: 10) }
+
+  before { subject.save }
```
Comment (Deepakdanger) [inline spec/models/post_spec.rb:6]: - [ ] After saving, kindly validate the values also.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @nunegabriel, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @nunegabriel , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :heavy_check_mark: 
Good effort so far :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**